torch/_inductor/codegen/cpp_micro_gemm.py

```diff
@@ -385,9 +424,12 @@ class CppMicroGemmFP32Vec(CppMicroGemm):
         }
 
         if constexpr (row == 0) {
-            {%- if input_dtype == torch.bfloat16 or input_dtype == torch.float16 %}
+            {%- if input2_dtype in [torch.bfloat16, torch.float16] %}
             auto b = VectorizedIn::loadu(B + k * ldb + col * VLEN, VLEN);
             vb[col] = at::vec::convert<{{compute_t}}>(b);
+            {%- elif input2_dtype == torch.int8 %}
+            auto b32 = at::vec::convert_to_int32<int8_t>(B + k * ldb + col * VLEN);
+            vb[col] = at::vec::convert<float>(b32);
```

Comment:
This might not be efficient to load 1/4 of the vector lanes per load.

Reply:
Thanks! I'll try using `at::vec::convert` with `VectorizedN`s as src & dst.